composer.json

```diff
@@ -60,8 +64,8 @@
         }
     },
     "scripts": {
-        "lint": "php-cs-fixer fix -v",
-        "test:lint": "php-cs-fixer fix -v --dry-run",
+        "lint": "vendor/bin/pint",
```

Comment:
Seems a bit odd having Pint here, when it's not a dev dependency 🤔

Reply:
Great 👍🏻 Looks like there's just one file to fix with Pint and then CI should all be green. 🥳